WcaOnRails/app/views/competitions/index.html.erb

```diff
@@ -20,7 +20,3 @@
     </div>
   </div>
 </div>
-
-<script>
-  $('#competitions-<%= params[:display] == "list" ? "map" : "list" %>').hide();
-</script>
```

Comment:
One bug that I see here. You removed this piece of code but didn't add the appropriate class after an html request.